sanity-io/structured-content-2022
# Editorial Articles: render `relatedTo` entities, `_createdAt`, `_updatedAt`

# Editorial Articles: render `relatedTo` entities, `_createdAt`, `_updatedAt`

## Intent

Renders `relatedTo` (people, sessions, venues) and `authors` entities in Editorial Article pages.

Fixes [this](https://app.shortcut.com/sanity-io/story/16417/editorial-article-front-end) Shortcut Story.

## Description

- Conditionally hiding the pipe (`|`) separating the `_createdAt` and `_updatedAt` wasn't solved in the smoothest way. If you can think of a better solution, feel free to overwrite the current solution, @oyvind-stenhaug.
- The author/person thumbnails get completely blurry if we set the `size` to match that of the Figma sketches -- setting them to 4x of the expected dimensions fixes this blur :shrug: 
- @kmelve mentioned in today's sync that a separate "updated at"-field will be pushed. A separate follow-up PR can handle this minor change if this PR gets approved prior to that being in place.
- @oyvind-stenhaug I'm going to let you work your magic wrt. to the layout changes required to have the positioning of the `relatedTo` entities in large viewports match the Figma sketches if you're OK with that. (Please let me know if you'd rather have me fix it. I don't mind at all, just assuming it'll be faster for you to do it. :slightly_smiling_face:)

## Testing this PR

1. Open this PR's Preview Deploy and navigate to /scholarships
2. Verify that both `authors` and `relatedTo` entities are rendered in accordance with the Figma specifications
    * NB! We're using the `staging` dataset now; I've added these associations for the Scholarships Editorial Article there and not in production :slightly_smiling_face:
